jobbergate-api/pyproject.toml

```diff
@@ -92,8 +92,14 @@ ignore = "W503,D200,D106,D402"
 
 
 [tool.pytest.ini_options]
-minversion = "6.0"
-addopts = ["--random-order", "--cov=jobbergate_api"]
+minversion = "7.0"
+addopts = [
+    "--random-order",
+    "--cov=jobbergate_api",
+    "--cov-report=term-missing",
+    "--cov-fail-under=90",
```

Comment:
Why use command line arguments here instead of using the configuration blocks you removed below?

Reply:
Just for consistency across the components. The three others define them using command-line arguments, so I changed the API to do the same.

Reply:
I think it would actually be better to go the other way and start using configuration as opposed to command line arguments for the other sub-projects as well.

Reply:
Following your suggestions, I changed the API and other components to use configuration blocks.

As a side note, I found no way to set the report format on the blocks. If I try `format="xml"` it complains when loading the config that the value should be boolean. It is probably a bug in the package since `format=true` results in an error when writing the report file because the format `true` is not supported.

We will stick to `--cov-report=xml:tests/coverage.xml` for now.